### ISSUE TITLE
a11y: va-avatar

### ISSUE THREAD

**Issue by kushich** [epicmaxco/vuestic-ui#3261]
- [x] step 1: investigate a11y guides for this component;
- [x] step 2: discuss with team;
- [ ] step 3: implement required functionality;
- [ ] step 4: add docs page section about a11y.

**Comment by kushich** [comment]
In our case avatar-group has navigation by TAB button, but it shouldn't.
<img width="321" alt="Screenshot 2023-03-28 at 11 34 20" src="https://user-images.githubusercontent.com/45711229/228201621-05cb67be-b827-4875-9778-be23c38a904b.png">
Also I'm not sure about `aria-hidden` and `aria-live`. 

`aria-hidden`: For avatars without some text description we should have some info about its content. So there should be visible content for screen readers ([ref](https://mineral-ui.netlify.app/components/avatar)).
<img width="590" alt="Screenshot 2023-03-28 at 14 12 03" src="https://user-images.githubusercontent.com/45711229/228218835-285a7522-5ebf-489d-b2ad-e2e916db7135.png">


  `aria-live`:
  > aria-live="polite" is used. Any region which receives updates that are important for the user to receive, but not so rapid as to be annoying, should receive this attribute. The screen reader will speak changes whenever the user is idle.
  
  Not sure that it's important for user to know about avatar. Or at least should be restricted for avatars with src prop. (**_BTW what is the purpose of `loading` prop for our avatar component?_**)

**Comment by asvae** [comment]
Maybe we shouldn't use aria-live at all here, loading is just not imporant enough to spam user.
